apps/codelab/src/app/codelabs/angular/component-tree/components-hierarchy-svg/components-hierarchy-svg.component.ts

```diff
@@ -0,0 +1,10 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'slides-tree-hierarchy-svg',
+  templateUrl: './components-hierarchy-svg.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ComponentsHierarchySvgComponent {
```

Comment:
Can we make sure all the file names match component name, match the selector?

Reply:
What exactly do you mean

Reply:
Done, thanks